projects/kit/components/rating/rating.template.html

```diff
@@ -0,0 +1,41 @@
+<label>
+    <ng-content></ng-content>
+
+    <div class="t-host">
+        <div class="t-interactive">
+            <tui-svg
+                *tuiRepeatTimes="let index of max"
+                class="t-icon-filled"
+                [src]="iconFilled"
+                (click)="setRateByReverseIndex(index)"
+            ></tui-svg>
+        </div>
+
+        <div #wrapper class="t-wrapper" [style.width.%]="percent">
+            <tui-svg
+                *tuiRepeatTimes="let index of max"
+                class="t-icon-filled"
+                [src]="iconFilled"
+            ></tui-svg>
+        </div>
+
+        <tui-svg
+            *tuiRepeatTimes="let item of max"
+            class="t-icon-normal"
+            [src]="iconNormal"
+        ></tui-svg>
+
+        <input
+            #focusableElement
+            type="range"
+            step="1"
+            class="t-input"
+            [min]="min"
+            [max]="max"
+            [ngModel]="value"
+            [tuiFocusable]="isFocusable"
+            [style.height.px]="wrapper.offsetHeight"
```

Comment:
Is this necessary? Looks like it can just be 100% since `t-host` is position: relative and its height is dependent on icons already. Besides we do not allow to change icons size so this can just be fixed at 1.5 rem.

Reply:
fixed